Components/AccordionVertical/script.js

```diff
@@ -0,0 +1,130 @@
+import $ from 'jquery'
+
+class AccordionVertical extends window.HTMLDivElement {
+  constructor (self) {
+    self = super(self)
+    self.$ = $(self)
+    self.setOptions()
+    self.resolveElements()
+    return self
+  }
+
+  setOptions () {
+    this.aria = {
+      hidden: 'aria-hidden',
+      controls: 'aria-controls',
+      expanded: 'aria-expanded'
+    }
+    this.class = {
+      expanded: 'accordionVertical-item-isExpanded',
+      hidden: 'accordionVertical-item-isHidden',
+      contentIsSizing: 'accordionVertical-content-isSizing'
+    }
+    this.isMultiselectable = this.$.attr('aria-multiselectable')
+  }
+
+  resolveElements () {
+    this.$item = $('.accordionVertical-item', this)
+    this.$button = $('.accordionVertical-btn[aria-controls]', this)
+    this.$content = $('.accordionVertical-content', this)
+  }
+
+  connectedCallback () {
+    this.$.on('click', this.$button.selector, this.toggleButton)
+    $(window).on('load resize', this.updateExpandedItemHeight)
```

Comment:
Actually no, would this be better? I need to call it once when the component initialises.

```js
this.updateExpandedItemHeight()
$(window).on('resize', this.updateExpandedItemHeight)
```